src/shared/lib/NumberUtils.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { floatValueIsNA } from "./NumberUtils";
+import { assert } from 'chai';
+
+describe('NumberUtils', () => {
+    
+    describe('floatValueIsNA', () => {
+        const naValue = Number.MIN_VALUE;
```

Comment:
I believe JAVA_MIN_FLOAT should be tested too. What about the negative value?

Reply:
Wanna follow up on this.

Reply:
This function is no longer in use. It is being dropped.